R/topics.R

```diff
@@ -92,9 +92,20 @@ match_env <- function(topics) {
         purrr::map_lgl(any)
 
       which(match & is_public(internal))
+    },
+    none_of_concepts = function(x, internal = FALSE) {
+      nomatch <- topics$concepts %>%
```

Comment:
I think this would be easier to follow if you just inverted the results of `has_concept()` but maybe that doesn't work?

Reply:
Two reasons: 
1) Non-public would be included (but could of course be filtered out) 
2) Multiple concepts should be supported

The definition happens inside that list which makes it impossible to use the `has_concept()`. This is actually what the `funs[['none_of_concepts']]` refers to above: an artefact of implementing it this way previously.

Do you have any preference: define `none_of_concepts` (or `lacks_concepts()`) inside the list or afterwards exploiting `has_concept()`?

Reply:
Ah ok, makes sense.